src/Simulation/QSharpFoundation/Convert/Convert.qs

```diff
@@ -61,48 +60,47 @@ namespace Microsoft.Quantum.Convert {
     function BoolArrayAsBigInt(a : Bool[]) : BigInt {
         body intrinsic;
     }
-     
+
     /// # Summary
     /// Converts a given boolean value to an equivalent string representation.
     ///
     /// # Remarks
     /// See [C# Convert.ToString](https://docs.microsoft.com/dotnet/api/system.convert.tostring?view=netframework-4.7.1#System_Convert_ToString_System_Boolean_) for more details.
     function BoolAsString(a : Bool) : String {
-        body intrinsic;
+        return $"{a}";
     }
-    
-    
+
+
     /// # Summary
     /// Converts a given double-precision floating-point number to an equivalent string representation.
     ///
     /// # Remarks
     /// See [C# Convert.ToString](https://docs.microsoft.com/dotnet/api/system.convert.tostring?view=netframework-4.7.1#System_Convert_ToString_System_Double_) for more details.
     function DoubleAsString(a : Double) : String {
-        body intrinsic;
+        return $"{a}";
```

Comment:
@bettinaheim: Bettina, does this change mean that `%String* @__quantum__rt__double_to_string(double %val)` and other to string conversion functions can be removed from the QIR spec and from the runtime?

Reply:
Unfortunately, at least not yet. I don't have anything convenient to call in llvm directly and I would need to look in detail into how to do this using string initialization only. For now, this interpolated string will precisely result in a call to __quantum__rt__double_to_string and this change merely removed the redundancy in the QIR generation that would otherwise have required to handle the function call and the string interpolation separately.